target/linux/mvebu/cortexa9/base-files/etc/uci-defaults/03_wireless

```diff
@@ -14,9 +14,10 @@ linksys,wrt1200ac|\
 linksys,wrt1900ac-v1|\
 linksys,wrt1900ac-v2|\
 linksys,wrt1900acs|\
+linksys,wrt3200acm|\
 linksys,wrt32x)
 	SKU=$(strings /dev/mtd3|sed -ne 's/^cert_region=//p')
-	WIFIMAC2G=$(macaddr_add $(cat /sys/class/net/eth0/address) +1)
+	WIFIMAC2G=$(macaddr_add $(mtd_get_mac_ascii devinfo hw_mac_addr) +1)
```

Comment:
can you test with `ethaddr` instead of `hw_mac_addr` ?
or `mtd_get_mac_ascii u-boot-env ethaddr`
hw_mac_addr will seems to be not available if mvneta is compiled as a module

Reply:
@lucize Then this should also be changed in 02_network? (Particularly since board.d happens _before_ uci-defaults ...)

Reply:
Neither of those worked for me on the _mamba_, no error message to be seen. I recall _dengqf6_ having to fight some to get the MAC for this device in the mvebu:DSA PR.

Reply:
`fw_printenv | grep ethaddr` is not providing the correct mac whitin os ?

Reply:
It does provide the correct mac.